MC-33183: New In-store delivery feature. Updated existing information and added new topics.

danidelcar opened the pull request:
## Purpose of this pull request

<!-- REQUIRED Describe the goal and the type of changes this pull request covers. Tell us what changes are you making and why. -->

This pull request (PR) adds information for the In-store delivery feature. Updates existing pages with new information, as well as creates new topics that cover gaps from this new feature.

If Magento Commerce, please specify:

- [x] Commerce only?
- [ ] Commerce with B2B?

## Affected documentation pages

<!-- REQUIRED List HTML links for affected pages on https://docs.magento.com -->

- [inventory-sources-add](https://docs.magento.com/user-guide/catalog/inventory-sources-add.html)
- [inventory-sources](https://docs.magento.com/user-guide/catalog/inventory-sources.html)
- [Settings-other](https://docs.magento.com/user-guide/catalog/settings-other.html)
- Sources (new page)
- [Settings](https://docs.magento.com/user-guide/catalog/settings.html)
- catalog.yml (added entry)
- Sales.yml (added entry)
- [Inventory-management](https://docs.magento.com/user-guide/catalog/inventory-management.html)
- [Inventory-migration](https://docs.magento.com/user-guide/catalog/inventory-migration.html)
- [Shipping methods](https://docs.magento.com/user-guide/configuration/sales/shipping-methods.html)
- shipping-in-store-delivery (new page)
- [Methods-basic](https://docs.magento.com/user-guide/shipping/methods-basic.html)
- [Sales-emails](https://docs.magento.com/user-guide/configuration/sales/sales-emails.html)
- [Email-template-list](https://docs.magento.com/user-guide/marketing/email-template-list.html)
- [Checkout-step1-shipping](https://docs.magento.com/user-guide/sales/checkout-step1-shipping.html)
- [Order-processing](https://docs.magento.com/user-guide/sales/order-processing.html)
- [Mcom (modified title)](https://docs.magento.com/user-guide/mcom.html)
- Removed in-store oms topic (just a temp file that I included)

## Additional information

JIRA [MC-33183](https://jira.corp.magento.com/browse/MC-33183)
Preview build [#522](https://docs.magedevteam.com/522/user-guide/)

whatsnew
Added new topics and updated information for the [In-store Delivery method](https://docs.magento.com/user-guide/shipping/shipping-in-store-delivery.html) provided by Inventory Management.

danidelcar commented:
Hello @leslietilling @keharper @shrielenee @jeff-matthews @sidolov , created the PR, changed it to draft. I included files that were modified, or created during this PR. As well as the new images I included with this PR. 

@shrielenee added you as a reviewer so could take a look at the mcom.md file modified, related to the conversation we had in slack about title for order management.
@jeff-matthews included you as discussed during the 1:1, feel free to review too!.
@leslietilling & @keharper added you as peer review and experts in merchdocs, as discussed previously these days.
Most likely I can add you @sidolov once we have this peer review out of the way, what do you think?

I got 4 conflicting files that I can review today or with you later on in a meeting we have scheduled. @jeff-matthews set it up last week. Thanks!

danidelcar commented:
@leslietilling could not see how to create a preview branch for this PR, as I normally do for oms?, thanks!

shrielenee commented on src/catalog/inventory-sources.md:
```diff
@@ -2,9 +2,11 @@
 ---
 
-Sources represent locations storing and shipping your product stock. Any location with available stock and capable of order fulfillment can be added as a source such as warehouses, brick-and-mortar stores, distribution centers, and drop shippers.
+Sources represent locations storing and shipping your product stock. Any location with available stock and capable of order fulfillment can be added as a source such as warehouses, brick-and-mortar stores, distribution centers, pickup locations and drop shippers.
```
Add a serial comma

danidelcar replied:
done

shrielenee commented on src/mcom.md:
```diff
@@ -1,6 +1,6 @@
 ---
 ee_only: true
-title: MCOM
+title: Magento Commerce Order Management (MCOM)
```
This looks good! Did not review the whole PR.

danidelcar replied:
Thanks @shrielenee !, you were not supposed to, thanks!

danidelcar commented on Gemfile.lock:
```diff
@@ -52,9 +52,6 @@ GEM
     image_optim_pack (0.6.0.20200215)
       fspath (>= 2.1, < 4)
       image_optim (~> 0.19)
-    image_optim_pack (0.6.0.20200215-x86_64-darwin)
```
Hi @jeff-matthews do you remember what I was trying to show you during the 1:1?, this is what happens, when I run rake to generate the local preview I get the gemfile.lock like it has been modified. So what I did was run bundle install and I got this into the PR. How can we fix it?, thanks!

jeff-matthews replied:
@danidelcar, I believe this happens when you run `bundle update` instead of `bundle install`. You should revert changes to this file.

danidelcar replied:
I'll check, thanks

danidelcar commented:
Hi @sidolov , after discussing with @jeff-matthews I added you already, so you can review this PR 👌 . Thanks in advance!

leslietilling commented:
> @leslietilling could not see how to create a preview branch for this PR, as I normally do for oms?, thanks!

@danidelcar  If the branch is updated in the repo, you can use this jenkins job and specify the branch name in the public repo: https://devdocs.ci.corp.adobe.com/view/merchdocs/job/merchdocs-staging/

leslietilling commented on src/catalog/inventory-sources-add.md:
```diff
@@ -100,5 +117,7 @@ If editing the Default Source, you can edit all configurations except name and c
 |Postcode|(Required) The ZIP or postal code of the inventory source.|
+|**Pickup Location**| |
+|Frontend Name|The name of the Pickup Location for the source.|
+|Frontend Description|The description of the Pickup Location for the source. It can contain attached images.|
```
```suggestion
|Frontend Description|The description of the pickup location for the source. It can contain attached images.|
```

danidelcar replied:
done

leslietilling commented on src/catalog/inventory-sources.md:
```diff
@@ -2,9 +2,11 @@
 ---
 
-Sources represent locations storing and shipping your product stock. Any location with available stock and capable of order fulfillment can be added as a source such as warehouses, brick-and-mortar stores, distribution centers, and drop shippers.
+Sources represent locations storing and shipping your product stock. Any location with available stock and capable of order fulfillment can be added as a source such as warehouses, brick-and-mortar stores, distribution centers, pickup locations, and drop shippers.
```
```suggestion
Sources represent locations storing and shipping your product stock. Any location with available stock and capable of order fulfillment can be added as a source, such as a warehouse, brick-and-mortar store, distribution center, pickup location, and drop shipper.
```

danidelcar replied:
I agree with Kevin, deleted!, thanks!

leslietilling commented on src/catalog/inventory-sources.md:
```diff
@@ -2,9 +2,11 @@
-Sources are the physical locations where product inventory is managed and shipped for order fulfillment, or where services are available. These locations can include warehouses, brick-and-mortar stores, distribution centers, and drop shippers. You allocate inventory quantities to these sources, and Magento automatically aggregates the total salable products for your stocks. For large companies, a ...
+Sources are the physical locations where product inventory is managed and shipped for order fulfillment, or where services are available. These locations can include warehouses, brick-and-mortar stores, distribution centers, pickup locations, and drop shippers. You allocate inventory quantities to these sources, and Magento automatically aggregates the total salable products for your stocks. For  ...
+
+It is recommended to provide specific physical geographical locations when creating a source. That allows the _Distance Priority Algorithm_ to compare the location of the shipping destination address with the available source locations to determine the closest source to fulfil shipments. You can use google maps or offline calculations, which use geocodes. For more information on this _Distance Pr ...
```
```suggestion
It is recommended to provide specific physical geographical locations when creating a source. That allows the _Distance Priority Algorithm_ to compare the location of the shipping destination address with the available source locations to determine the closest source to fulfill shipments. You can use Google Maps or offline calculations, which use geocodes. For more information on this _Distance Priority Algorithm_, see [Configure Distance Priority Algorithm]({% link catalog/inventory-configure-distance-priority.md %}).
```

danidelcar replied:
done!

leslietilling commented on src/catalog/sources.md:
```diff
@@ -0,0 +1,22 @@
+title: Sources
+---
+
+The Sources section lists the sources that contain stock for a specific product. It is used to modify the stock quantity for each product in a source. For more information about **Sources**, see [Managing Sources]({% link catalog/inventory-sources.md %}).
```
```suggestion
The _Sources_ section lists the sources that contain stock for a specific product. It is used to modify the stock quantity for each product in a source. For more information about sources, see [Managing Sources]({% link catalog/inventory-sources.md %}).
```

danidelcar replied:
applied both, thanks!

leslietilling commented on src/catalog/sources.md:
```diff
@@ -0,0 +1,22 @@
+![]({% link images/images/sources_view.png %}){: .zoom}
+_Sources View_
+
+It is possible to assign it to a source with a specific delivery method, such as an In-store delivery method.
```
```suggestion
It is possible to assign it to a source with a specific delivery method, such as an in-store delivery method.
```

danidelcar replied:
true, does not make sense to have it here Kevin, I basically wanted to highlight that you can assign a source to a delivery method such as the in-store pickup delivery method

leslietilling commented on src/catalog/sources.md:
```diff
@@ -0,0 +1,22 @@
+|--- |--- |
+|Name|The unique name for a source.|
+|Source Status|Determines if the product is enabled or disabled in the catalog.|
+|Source Item Status|Determines the current availability of the product. Options:<br />**In Stock** - Makes the product available for purchase.<br />**Out of Stock** - Unless Backorders are activated, prevents the product from being available for purchase and removes the listing from the catalog.|
```
```suggestion
|Source Item Status|Determines the current availability of the product. Options:<br />**In Stock** - Makes the product available for purchase.<br />**Out of Stock** - Unless backorders are activated, prevents the product from being available for purchase and removes the listing from the catalog.|
```

danidelcar replied:
done!

leslietilling commented on src/catalog/sources.md:
```diff
@@ -0,0 +1,22 @@
+|Name|The unique name for a source.|
+|Source Status|Determines if the product is enabled or disabled in the catalog.|
+|Source Item Status|Determines the current availability of the product. Options:<br />**In Stock** - Makes the product available for purchase.<br />**Out of Stock** - Unless Backorders are activated, prevents the product from being available for purchase and removes the listing from the catalog.|
+|Qty|On-hand stock amounts for each source, or location, for the source.|
```
```suggestion
|Qty|On-hand stock amounts for each source, or location for the source.|
```

danidelcar replied:
done!

leslietilling commented on src/catalog/sources.md:
```diff
@@ -0,0 +1,22 @@
+|Source Status|Determines if the product is enabled or disabled in the catalog.|
+|Source Item Status|Determines the current availability of the product. Options:<br />**In Stock** - Makes the product available for purchase.<br />**Out of Stock** - Unless Backorders are activated, prevents the product from being available for purchase and removes the listing from the catalog.|
+|Qty|On-hand stock amounts for each source, or location, for the source.|
+|Notify Qty|An amount for the Notify for Quantity for this specific source if `Notify Quantity Use Default` is not selected.|
```
```suggestion
|Notify Qty|An amount for the _Notify for Quantity_ for this specific source if `Notify Quantity Use Default` is not selected.|
```

danidelcar replied:
done, thanks!

leslietilling commented on src/catalog/sources.md:
```diff
@@ -0,0 +1,22 @@
+|Source Item Status|Determines the current availability of the product. Options:<br />**In Stock** - Makes the product available for purchase.<br />**Out of Stock** - Unless Backorders are activated, prevents the product from being available for purchase and removes the listing from the catalog.|
+|Qty|On-hand stock amounts for each source, or location, for the source.|
+|Notify Qty|An amount for the Notify for Quantity for this specific source if `Notify Quantity Use Default` is not selected.|
+|Notify Qty Use Default|Indicates to use the default setting for Notify for Quantity in the product **Advanced Inventory** or global setting in Store configuration. For more information about **Advanced Inventory** for your product, see [Configure advanced product options]({% link catalog/inventory-product-stock-options.md %}).|
```
```suggestion
|Notify Qty Use Default|Indicates to use the default setting for _Notify for Quantity_ in the product Advanced Inventory or global setting in the Store configuration. For more information about advanced inventory settings for your product, see [Configure advanced product options]({% link catalog/inventory-product-stock-options.md %}).|
```

danidelcar replied:
thanks!

leslietilling commented on src/catalog/sources.md:
```diff
@@ -0,0 +1,22 @@
+|Qty|On-hand stock amounts for each source, or location, for the source.|
+|Notify Qty|An amount for the Notify for Quantity for this specific source if `Notify Quantity Use Default` is not selected.|
+|Notify Qty Use Default|Indicates to use the default setting for Notify for Quantity in the product **Advanced Inventory** or global setting in Store configuration. For more information about **Advanced Inventory** for your product, see [Configure advanced product options]({% link catalog/inventory-product-stock-options.md %}).|
+|Actions|**Unassign** - The source is not available for the product. You can use the **Assign Sources** action to make a source available for the product. For more information about **Assign Sources** options, see [Assigning Sources per Product]({% link catalog/inventory-product-sources.md %}).|
```
```suggestion
|Actions|For an assigned source, click **Unassign** to make the source not available for the product. For an unassigned source, click **Assign Sources** to make a source available for the product. For more information about Assign Sources options, see [Assigning Sources per Product]({% link catalog/inventory-product-sources.md %}).|
```

danidelcar replied:
done!

leslietilling commented on src/configuration/sales/delivery-methods.md:
```diff
@@ -66,6 +66,20 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
+|Field|[Scope]({% link configuration/scope.md %})|Description|
+|--- |--- |--- |
+|Enabled|Website|When enabled, In-Store Delivery appears as an option in the Estimate Shipping and Tax section of the shopping cart, and in the Shipping section during checkout. Options: Yes / No|
+|Method Name|Store View|A name that describes the method of calculation that is used to produce a shipping estimate. The method name appears next to the calculated estimated rate in the shopping cart. The default value is "In-store Delivery”.|
```
```suggestion
|Method Name|Store View|A name that describes the method of calculation that is used to produce a shipping estimate. The method name appears next to the calculated estimated rate in the shopping cart. The default value is `In-store Delivery`.|
```

danidelcar replied:
done, thanks!

leslietilling commented on src/configuration/sales/delivery-methods.md:
```diff
@@ -66,6 +66,20 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
+|Enabled|Website|When enabled, In-Store Delivery appears as an option in the Estimate Shipping and Tax section of the shopping cart, and in the Shipping section during checkout. Options: Yes / No|
+|Method Name|Store View|A name that describes the method of calculation that is used to produce a shipping estimate. The method name appears next to the calculated estimated rate in the shopping cart. The default value is "In-store Delivery”.|
+|Title|Store View|The name that is used for this shipping method during checkout.|
+|Price|Website|The price you charge the customer for a In-Store Delivery shipping.|
```
```suggestion
|Price|Website|The price you charge the customer for in-store delivery shipping.|
```

danidelcar replied:
makes sense the simplification by Kevin, thanks to you 2!

leslietilling commented on src/configuration/sales/delivery-methods.md:
```diff
@@ -66,6 +66,20 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
+|Title|Store View|The name that is used for this shipping method during checkout.|
+|Price|Website|The price you charge the customer for a In-Store Delivery shipping.|
+|Search Radius|Website|Radius for a pickup location search on the storefront checkout. Appears in `Km`.|
+|Displayed Error Message|Store View|A message that appears if a customer chooses In-Store Delivery, but for some reason the method is not available.|
```
```suggestion
|Displayed Error Message|Store View|A message that appears if a customer chooses in-store delivery, but for some reason the method is not available.|
```

danidelcar replied:
thanks Leslie!

leslietilling commented on src/configuration/sales/sales-emails.md:
```diff
@@ -125,6 +125,20 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
+
+|Field|[Scope]({% link configuration/scope.md %})|Description|
+|--- |--- |--- |
+|Enabled|Store View|When enabled, sends a transactional email for each Order Ready For Pickup in Store. Options include: Yes / No|
```
```suggestion
|Enabled|Store View|When enabled, sends a transactional email for each order that is ready for pickup in store. Options: Yes / No|
```

danidelcar replied:
done, thanks!

leslietilling commented on src/configuration/sales/sales-emails.md:
```diff
@@ -125,6 +125,20 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
+|--- |--- |--- |
+|Enabled|Store View|When enabled, sends a transactional email for each Order Ready For Pickup in Store. Options include: Yes / No|
+|Order Ready For Pickup Email Sender|Store View|Identifies the store contact that appears as the message sender. Default sender: General Contact|
+|Order Ready For Pickup Email Template|Store View|Identifies the template that is sent when an Order Ready For Pickup in Store. Default template: Order is Ready for Pickup|
```
```suggestion
|Order Ready For Pickup Email Template|Store View|Identifies the template that is used for the transactional email for each order that is ready for pickup in store. Default template: `Order is Ready for Pickup`|
```

danidelcar replied:
done, thanks you 2!

leslietilling commented on src/configuration/sales/sales-emails.md:
```diff
@@ -125,6 +125,20 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
+|Enabled|Store View|When enabled, sends a transactional email for each Order Ready For Pickup in Store. Options include: Yes / No|
+|Order Ready For Pickup Email Sender|Store View|Identifies the store contact that appears as the message sender. Default sender: General Contact|
+|Order Ready For Pickup Email Template|Store View|Identifies the template that is sent when an Order Ready For Pickup in Store. Default template: Order is Ready for Pickup|
+|Order Ready For Pickup Email Template for Guest|Store View|Identifies the template that is sent when an Order Ready For Pickup in Store for a guest. Default template: Order is Ready for Pickup for Guest|
```
```suggestion
|Order Ready For Pickup Email Template for Guest|Store View|Identifies the template that is used for the transactional email for each order that is ready for pickup in store for a guest. Default template: `Order is Ready for Pickup for Guest`|
```

danidelcar replied:
done, thanks!

leslietilling commented on src/configuration/sales/sales-emails.md:
```diff
@@ -125,6 +125,20 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
+|Order Ready For Pickup Email Sender|Store View|Identifies the store contact that appears as the message sender. Default sender: General Contact|
+|Order Ready For Pickup Email Template|Store View|Identifies the template that is sent when an Order Ready For Pickup in Store. Default template: Order is Ready for Pickup|
+|Order Ready For Pickup Email Template for Guest|Store View|Identifies the template that is sent when an Order Ready For Pickup in Store for a guest. Default template: Order is Ready for Pickup for Guest|
+|Send Order Ready For Pickup Email Copy To|Store View|Specifies the email address of anyone to receive a copy of an Order Ready For Pickup email. Separate multiple addresses with a comma.|
```
```suggestion
|Send Order Ready For Pickup Email Copy To|Store View|Specifies the email address of anyone to receive a copy of an _Order Ready For Pickup_ email. Separate multiple addresses with a comma.|
```

danidelcar replied:
thanks!

leslietilling commented on src/configuration/sales/sales-emails.md:
```diff
@@ -125,6 +125,20 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
+|Field|[Scope]({% link configuration/scope.md %})|Description|
+|--- |--- |--- |
+|Enabled|Store View|When enabled, sends a transactional email for each Order Ready For Pickup in Store. Options include: Yes / No|
+|Order Ready For Pickup Email Sender|Store View|Identifies the store contact that appears as the message sender. Default sender: General Contact|
```
```suggestion
|Order Ready For Pickup Email Sender|Store View|Identifies the store contact that appears as the message sender. Default sender: `General Contact`|
```

danidelcar replied:
done!

leslietilling commented on src/configuration/sales/sales-emails.md:
```diff
@@ -125,6 +125,20 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
+|Order Ready For Pickup Email Template|Store View|Identifies the template that is sent when an Order Ready For Pickup in Store. Default template: Order is Ready for Pickup|
+|Order Ready For Pickup Email Template for Guest|Store View|Identifies the template that is sent when an Order Ready For Pickup in Store for a guest. Default template: Order is Ready for Pickup for Guest|
+|Send Order Ready For Pickup Email Copy To|Store View|Specifies the email address of anyone to receive a copy of an Order Ready For Pickup email. Separate multiple addresses with a comma.|
+|Send Order Ready For Pickup Email Copy Method|Store View|Indicates the email method used to send the copy. Options include: <br/>**Bcc** - Sends a blind courtesy copy by including the recipient in the header of the same email that is sent to the customer. The BCC recipient is not visible to the customer. <br/>**Separate Email** - Sends the copy as a separate email.|
```
```suggestion
|Send Order Ready For Pickup Email Copy Method|Store View|Indicates the email method used to send the copy. Options: <br/>**Bcc** - Sends a blind courtesy copy by including the recipient in the header of the same email that is sent to the customer. The BCC recipient is not visible to the customer. <br/>**Separate Email** - Sends the copy as a separate email.|
```

danidelcar replied:
thanks!

leslietilling commented on src/sales/checkout-step1-shipping.md:
```diff
@@ -2,7 +2,9 @@
 ---
 
-The first step of the checkout process is for the customer to complete the shipping address information, and to choose the shipping method. If the customer has an account, the shipping address is entered automatically, but can be changed if needed.
+The first step of the checkout process is for the customer to complete the shipping address information, and to choose the delivery method. For more information, see [delivery methods]({% link configuration/sales/delivery-methods.md %}).
```
```suggestion
The first step of the checkout process is for the customer to complete the shipping address information and choose the delivery method. For more information, see [delivery methods]({% link configuration/sales/delivery-methods.md %}).
```

danidelcar replied:
done!

leslietilling commented on src/sales/checkout-step1-shipping.md:
```diff
@@ -11,6 +13,20 @@ The progress bar at the top of the page follows each step of the checkout proces
+![]({% link images/images/select-in-store-delivery.png %}){: .zoom}
+_Checkout: Shipping Method_
+
+This shipping method allows for a pickup location to be selected. If the customer has an account, the default pickup location address is entered automatically, but can be changed if needed. Click **Select other** and a list of the available pickup locations appear.
```
```suggestion
This shipping method allows for a pickup location to be selected. If the customer has an account, the default pickup location address is entered automatically, but can be changed if needed. When the customers clicks **Select other**, a list of the available pickup locations appears.
```

danidelcar replied:
thanks a lot!

leslietilling commented on src/sales/checkout-step1-shipping.md:
```diff
@@ -11,6 +13,20 @@ The progress bar at the top of the page follows each step of the checkout proces
+![]({% link images/images/select-in-store-delivery-populated.png %}){: .zoom}
+_Checkout: Default Pickup Location_
+
+For more information about how to enable this shipping method, see [In-store Delivery]({% link shipping/shipping-in-store-delivery.md %}) feature.
```
```suggestion
For more information about how to enable this method, see [In-store Delivery]({% link shipping/shipping-in-store-delivery.md %}).
```

danidelcar replied:
done

leslietilling commented on src/shipping/methods-basic.md:
```diff
@@ -12,6 +12,9 @@ Magento supports a wide range of delivery methods, as well as shipping carriers.
 
-If you are using [Inventory Management]({% link catalog/inventory.md %}), you can also set up Sources to provide in-store delivery options for your customers.
\ No newline at end of file
+If you are using [Inventory Management]({% link catalog/inventory.md %}), you can also set up Sources to provide in-store delivery options for your customers.
```
```suggestion
If you are using [Inventory Management]({% link catalog/inventory.md %}), you can also set up sources to provide in-store delivery options for your customers.
```

danidelcar replied:
done!

leslietilling commented on src/shipping/shipping-in-store-delivery.md:
```diff
@@ -0,0 +1,44 @@
+title: In-store Delivery
+---
+
+The In-store delivery method allows to select a source to be used as a pickup location during the checkout of your order.
```
```suggestion
With the in-store delivery method,  you can select a source to be used as a pickup location during the checkout.
```

danidelcar replied:
done!, and I agree with you Kevin, this is something we can raise with someone? (ENG or Product?), and then we can work on applying necessary changes?. Count me in!

leslietilling commented on src/shipping/shipping-in-store-delivery.md:
```diff
@@ -0,0 +1,44 @@
+The In-store delivery method allows to select a source to be used as a pickup location during the checkout of your order.
+
+![]({% link images/images/luma-in-store-example.png %}){: .zoom}
+[_Example of a In-store delivery method during checkout_]({% link sales/checkout-step1-shipping.md %})
```
```suggestion
[_In-store Delivery Method at Checkout_]({% link sales/checkout-step1-shipping.md %})
```

danidelcar replied:
done!

leslietilling commented on src/shipping/shipping-in-store-delivery.md:
```diff
@@ -0,0 +1,44 @@
+![]({% link images/images/luma-in-store-example.png %}){: .zoom}
+[_Example of a In-store delivery method during checkout_]({% link sales/checkout-step1-shipping.md %})
+
+## To set up In-store Delivery
```
```suggestion
## Set up In-store Delivery
```

danidelcar replied:
done!

leslietilling commented on src/shipping/shipping-in-store-delivery.md:
```diff
@@ -0,0 +1,44 @@
+
+1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
+
+1. In the _Sales_ section in the left panel, choose **Delivery Methods**.
```
```suggestion
1. In the left panel, expand **Sales** and choose **Delivery Methods**.
```

danidelcar replied:
reviewed!

leslietilling commented on src/shipping/shipping-in-store-delivery.md:
```diff
@@ -0,0 +1,44 @@
+
+1. Set **Enabled** to `Yes`.
+
+1. If necessary, clear the **Use system value** checkbox to edit for any field to be edited.
```
```suggestion
   {:.bs-callout-info}
   If needed, clear the **Use system value** checkbox to change the default for any field.
```

danidelcar replied:
nice, thanks!

leslietilling commented on src/shipping/shipping-in-store-delivery.md:
```diff
@@ -0,0 +1,44 @@
+
+1. Enter the **Search Radius** in kilometers for store pickup location search on storefront checkout.
+
+1. In the **Displayed Error Message** box, type the message that appears if In-Store Delivery becomes unavailable. ()
```
```suggestion
1. For **Displayed Error Message**, enter the message that appears if In-Store Delivery becomes unavailable.
```

danidelcar replied:
done!

leslietilling commented on src/shipping/shipping-in-store-delivery.md:
```diff
@@ -0,0 +1,44 @@
+    {:.bs-callout-info}
+    The default message is `In-Store Delivery is not available. To use this delivery method, please contact us.`
+
+1. Click **Save Config**.
```
```suggestion
1. Click <span class="btn">Save Config</span>.
```

danidelcar replied:
modified!, thanks!

leslietilling commented:
@danidelcar  After you complete editorial updates, please add a "What's New" item to the PR description. I can do the final approval to merge first thing tomorrow.

keharper commented on src/shipping/shipping-in-store-delivery.md:
```diff
@@ -0,0 +1,44 @@
+
+## To set up In-store Delivery
+
+The first step is to complete the default settings for In-store Delivery. You can complete this step without changing the scope of the configuration.
```
Rewrite this paragraph. 

Enable In-store Delivery and optionally change the default values.

danidelcar replied:
done

keharper commented on src/shipping/shipping-in-store-delivery.md:
```diff
@@ -0,0 +1,44 @@
+
+1. If necessary, clear the **Use system value** checkbox to edit for any field to be edited.
+
+1. Enter the **Method Name** that you want to appear as a label next to the calculated rate in the shopping cart.
```
I provided more detail earlier in this PR. Consider adapting that text for this entire procedure.

danidelcar replied:
modified, thanks!

keharper commented on src/shipping/shipping-in-store-delivery.md:
```diff
@@ -0,0 +1,44 @@
+    {:.bs-callout-info}
+    The default title is `In-Store Pickup Delivery`.
+
+1. Enter the **Price** that you want to charge for In-Store Delivery shipping.
```
```suggestion
1. Enter the **Price**  to charge customers for the in-store pickup service.
```

danidelcar replied:
done!

danidelcar commented on src/catalog/in-store-pickup.md:
```diff
@@ -0,0 +1,7 @@
+title: In-Store Pickup
+---
+
+In-store pickup (ISPU) functionality allows a customer to place their order online and collect the order items from a physical store. The Order Management System (OMS) provides a web-based interface to allow the store associate to pick and prepare the items for customer collection.
```
this was just a temp in the merchdocs page, I used it to not be navigating to oms to compare how the ispu feature was used and described there. I removed it now, apologies I thought I removed it before creating the draft PR.

danidelcar commented on src/catalog/in-store-pickup.md:
```diff
@@ -0,0 +1,7 @@
+
+In-store pickup (ISPU) functionality allows a customer to place their order online and collect the order items from a physical store. The Order Management System (OMS) provides a web-based interface to allow the store associate to pick and prepare the items for customer collection.
+
+Exceptions, such as items not able to be picked, can be handled simply and cleanly from this interface. The UI employs responsive design, which is a mobile-first methodology (meaning it can be used from a tablet device, such as an iPad, or a mobile phone).
```
see previous comment, that was a temp page, that I already removed

danidelcar commented on src/catalog/inventory-migration.md:
```diff
@@ -16,6 +16,8 @@ When first installing or upgrading to Magento 2.3.x Open Source, Commerce, or Co
 
    - All existing products in your catalog assign to the Default Source. Until you add new sources, the product interface does not change. If you only ship products from one location, you will not see additional differences for sources. You can create custom [sources]({% link catalog/inventory-sources-add.md %}) and [assign quantities]({% link catalog/inventory-manage-inventory-quantities.md %})  ...
 
+   - You can configure a source as a **Pickup Location** and [assign quantities]({% link catalog/inventory-manage-inventory-quantities.md %}) for that source.
```
done

danidelcar commented on src/catalog/inventory-migration.md:
```diff
@@ -36,13 +38,13 @@ When first installing or upgrading to Magento 2.3.x Open Source, Commerce, or Co
+## Magento Commerce Order Management and Inventory Management
 
-If you use [Magento Order Management][1] in your Magento implementation to manage, sell, and fulfill inventory from any sales channel, you need to fully disable the [Inventory Management extensions]({% link magento/magento-extensions.md %}). These modules provide all Inventory Management features to Magento, including Single and Multi Source management, stocks, reservations, and more.
+If you use [Magento Commerce Order Management (MCOM)][1] in your Magento implementation to manage, sell, and fulfill inventory from any sales channel, you need to fully disable the [Inventory Management extensions]({% link magento/magento-extensions.md %}). These modules provide all Inventory Management features to Magento, including Single and Multi Source management, stocks, reservations, and m ...
```
done

danidelcar commented on src/catalog/inventory-sources-add.md:
```diff
@@ -34,6 +36,10 @@ If editing the Default Source, you can edit all configurations except name and c
 
+    - If this inventory source is a Pickup Location, set to `Yes`.
+
+      The Default Source cannot be used as a pickup location for a In-store pickup delivery method.
```
done, thanks!

danidelcar commented on src/catalog/inventory-sources-add.md:
```diff
@@ -62,6 +68,17 @@ If editing the Default Source, you can edit all configurations except name and c
 
+1. Expand ![]({% link images/images/btn-expand.png %}) the **Pickup Location** section and do the following:
+
+    - Choose the **Frontend Name** of the **Pickup Location**.
```
thanks!

danidelcar commented on src/catalog/inventory-migration.md:
```diff
@@ -36,13 +38,13 @@ When first installing or upgrading to Magento 2.3.x Open Source, Commerce, or Co
+If you use [Magento Commerce Order Management (MCOM)][1] in your Magento implementation to manage, sell, and fulfill inventory from any sales channel, you need to fully disable the [Inventory Management extensions]({% link magento/magento-extensions.md %}). These modules provide all Inventory Management features to Magento, including Single and Multi Source management, stocks, reservations, and m ...
 
-We may support integrations between Magento Inventory Management (also known as MSI) and Magento Order Management in a later release.
+We may support integrations between Magento Inventory Management (also known as MSI) and MCOM in a later release.
```
true, done!

danidelcar commented on src/catalog/inventory-sources-add.md:
```diff
@@ -25,6 +25,8 @@ If editing the Default Source, you can edit all configurations except name and c
 
     - If this inventory source is ready to use, set **Is Enabled** to `Yes`.
 
+      The Default Source must be enabled. A default source is required for single source merchants and product migration.
```
allright, done!

danidelcar commented on src/catalog/inventory-sources-add.md:
```diff
@@ -62,6 +68,17 @@ If editing the Default Source, you can edit all configurations except name and c
+
+    - Choose the **Frontend Name** of the **Pickup Location**.
+
+    - Enter a **Frontend Description** of the **Pickup Location**.
```
Added, thanks!

danidelcar commented on src/catalog/inventory-sources-add.md:
```diff
@@ -100,5 +117,7 @@ If editing the Default Source, you can edit all configurations except name and c
 |Street|The street address of the inventory source.|
 |Postcode|(Required) The ZIP or postal code of the inventory source.|
+|**Pickup Location**| |
+|Frontend Name|The name of the Pickup Location for the source.|
```
thanks!

danidelcar commented on src/catalog/settings-other.md:
```diff
@@ -30,6 +30,7 @@ _Other Settings_{:.ee-only}
 
 | Section | Description |
 |----------
+| [Sources]({% link catalog/sources.md %}) | This section lists the sources that contain stock for the product. Allows you to modify the quantity in each source and availability for a specific delivery method. |
```
modified, thanks!

danidelcar commented on src/configuration/sales/delivery-methods.md:
```diff
@@ -66,6 +66,20 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
+
+|Field|[Scope]({% link configuration/scope.md %})|Description|
+|--- |--- |--- |
+|Enabled|Website|When enabled, In-Store Delivery appears as an option in the Estimate Shipping and Tax section of the shopping cart, and in the Shipping section during checkout. Options: Yes / No|
```
done!

danidelcar commented on src/configuration/sales/delivery-methods.md:
```diff
@@ -66,6 +66,20 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
+|Field|[Scope]({% link configuration/scope.md %})|Description|
+|--- |--- |--- |
+|Enabled|Website|When enabled, In-Store Delivery appears as an option in the Estimate Shipping and Tax section of the shopping cart, and in the Shipping section during checkout. Options: Yes / No|
+|Method Name|Store View|A name that describes the method of calculation that is used to produce a shipping estimate. The method name appears next to the calculated estimated rate in the shopping cart. The default value is "In-store Delivery”.|
```
done!

danidelcar commented on src/configuration/sales/delivery-methods.md:
```diff
@@ -66,6 +66,20 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
+|Method Name|Store View|A name that describes the method of calculation that is used to produce a shipping estimate. The method name appears next to the calculated estimated rate in the shopping cart. The default value is "In-store Delivery”.|
+|Title|Store View|The name that is used for this shipping method during checkout.|
+|Price|Website|The price you charge the customer for a In-Store Delivery shipping.|
+|Search Radius|Website|Radius for a pickup location search on the storefront checkout. Appears in `Km`.|
```
modified, thanks!

danidelcar commented on src/configuration/sales/delivery-methods.md:
```diff
@@ -66,6 +66,20 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
+|Title|Store View|The name that is used for this shipping method during checkout.|
+|Price|Website|The price you charge the customer for a In-Store Delivery shipping.|
+|Search Radius|Website|Radius for a pickup location search on the storefront checkout. Appears in `Km`.|
+|Displayed Error Message|Store View|A message that appears if a customer chooses In-Store Delivery, but for some reason the method is not available.|
```
modified, thanks!

danidelcar commented on src/configuration/sales/sales-emails.md:
```diff
@@ -125,6 +125,20 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
+
+|Field|[Scope]({% link configuration/scope.md %})|Description|
+|--- |--- |--- |
+|Enabled|Store View|When enabled, sends a transactional email for each Order Ready For Pickup in Store. Options include: Yes / No|
```
thanks!

danidelcar commented on src/sales/checkout-step1-shipping.md:
```diff
@@ -11,6 +13,20 @@ The progress bar at the top of the page follows each step of the checkout proces
 
+## Select In-store Delivery
+
+A new tab appears during checkout if the in-store delivery is enabled.
```
Added, thanks!

danidelcar commented on src/sales/checkout-step1-shipping.md:
```diff
@@ -11,6 +13,20 @@ The progress bar at the top of the page follows each step of the checkout proces
+![]({% link images/images/select-in-store-delivery.png %}){: .zoom}
+_Checkout: Shipping Method_
+
+This shipping method allows for a pickup location to be selected. If the customer has an account, the default pickup location address is entered automatically, but can be changed if needed. Click **Select other** and a list of the available pickup locations appear.
```
reviewed!

danidelcar commented on src/catalog/inventory-sources-add.md:
```diff
@@ -34,6 +36,10 @@ If editing the Default Source, you can edit all configurations except name and c
       ![]({% link images/images/stores-inventory-source-general.png %}){: .zoom}
       _General_
 
+    - If this inventory source is a Pickup Location, set to `Yes`.
```
done!

danidelcar commented:
Hello @leslietilling & @keharper . Reviewed all requested changes, generating new preview branch for this PR (522). Modified description and added whatsnew too. Only thing I am missing is to review the issue I had with the gemfile.lock I discussed with @jeff-matthews yesterday.

leslietilling commented:
running tests
